packages/ui/src/theme/css/component/select.scss

```diff
@@ -61,4 +63,15 @@
       --amplify-components-select-large-padding-inline-end
     );
   }
+
+  &[multiple],
+  &[size] {
+    overflow: auto;
+    padding: 0.25rem 1rem;
```

Comment:
Lets add tokens for these padding values. And the option padding below (we can sync on that if needed).